pkg/kubelet/dockershim/docker_service.go

```diff
@@ -76,7 +77,8 @@ type DockerLegacyService interface {
 }
 
 type dockerService struct {
-	client dockertools.DockerInterface
+	client                 dockertools.DockerInterface
+	podInfraContainerImage string
```

Comment:
Rename the field to `podSandboxImage`. We should not use the "infra container" name anymore in this package.